### ISSUE TITLE
Problem bug

### ISSUE THREAD

**Issue by SylvanFranklin** [rjain37/typstracer#5]
### Problem Title

Definition of derivative

### Problem Description

<img width="1087" alt="Screenshot 2024-12-21 at 8 15 49 PM" src="https://github.com/user-attachments/assets/b9b16ddc-f590-4a6e-a462-e4854423aa11" />

This is not identifying for some reason

### Typst Code

I lost the page sadly but it is in the image.

**Comment by rjain37** [comment]
That's weird. I tried your response exactly on my end and it seemed to work. Any ideas what the issue may stem from?

**Comment by SylvanFranklin** [comment]
Yeah super weird no idea if we have the same input, could it possibly be whitespace or is that ignored?
